1-callbacks/solution.js

```diff
@@ -2,7 +2,7 @@
 INSTRUCTIONS
 
 1. create an array that contains 5 names, include 1 or more of the allowed usernames located in validate-user.js
-2. iterate the array, keep an eye on performance, and validate every username with the function exported in validate-user.js
+2. iterate the array, keep an eye on performance, afirstnd validate every username with the function exported in validate-user.js
```

Comment:
While creating a PR, you should check if only your changes are being added. Taking this case as an example, this part was changed by mistake I guess, so you can fix this up, or maybe if this change was made intentionally, you can explain to your team why this change is being addressed in your ticket 👍